modules/swagger-codegen/src/main/java/io/swagger/codegen/CodegenConstants.java

```diff
@@ -62,4 +62,10 @@
     public static final String OPTIONAL_ASSEMBLY_INFO = "optionalAssemblyInfo";
     public static final String OPTIONAL_ASSEMBLY_INFO_DESC = "Generate AssemblyInfo.cs (Default: true).";
 
+    public static final String OPTIONAL_PROJECT_FILE = "optionalProjectFile";
+    public static final String OPTIONAL_PROJECT_FILE_DESC = "Generate {PackageName}.csproj (Default: true).";
```

Comment:
Correct, the default should be false. Initially I had it default to true but forgot to change the description. I can fix that.